### PR TITLE
always refresh the param list based on selection

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher this should fix the issue. ive testing some plots and things still work

### CODE REVIEW

**Comment by birdage** [comment]
screen shot 2015-12-09 at 11 38 25 am

**Comment by DanielJMaher** [comment]
@FBRTMaka @Bobfrat

**Comment by DanielJMaher** [comment]
@Bobfrat Quick Review and merge?

File: ooiui/templates/science/plotting.html [hunk 1]
```diff
@@ -535,11 +535,10 @@
       parameterhtml += "</optgroup>"
 
       $("div#yvar0-selection-default > div.form-group > select").html(parameterhtml);
-      if (refreshYVar1){
-        $("div#yvar1-selection > div.form-group > select").html(parameterhtml);
-      }
```
Comment (Bobfrat) [inline ooiui/templates/science/plotting.html]: @birdage @DanielJMaher this was in here for Interpolated plotting so I think removing it breaks that plot

Comment (birdage) [inline ooiui/templates/science/plotting.html]: @Bobfrat we just always refresh the list now as before it was not getting refreshed for the other plot types

**Comment by Bobfrat** [comment]
Looks good

**Pull request merged**